### PR TITLE
Cleanup and remove manual-test folder from grunt exclude lists

### PR DESCRIPTION
**Pull request by KaptenJansson**
* I've now cleaned up remaining TODO's.
* Fixed broken functionality (multiple cameras) in multiple-peerconnections.
* Removed redundant constraints page (nicer one exists in demos)
* Removed manual-test folder from Gruntfile.js exclude filter
* Move inline javascript to the end of the page before closing body tag
* Added 'use strict' where missing
* Named non public functions accordingly (appended "_")

### CODE REVIEW

**Comment by KaptenJansson** [comment]
I will do JSCS separately after this has landed now we know that JSCS is working correctly.

**Comment by KaptenJansson** [comment]
just noticed it cannot be merged automagically, will take a look.

**Comment by KaptenJansson** [comment]
I've now updated all files to comply with both JSCS and JSHINT. Removed links to the constraints page.

File: samples/web/content/manual-test/multiple-video/index.html [hunk 1]
```diff
@@ -68,5 +47,25 @@
           autoplay="autoplay"></video></td>
     </tr>
   </table>
+  <script src="../../../js/adapter.js"></script>
```
Comment (andresusanopinto) [inline samples/web/content/manual-test/multiple-video/index.html]: why move scripts into body?

Comment (KaptenJansson) [inline samples/web/content/manual-test/multiple-video/index.html]: Due to Sam's comment in https://github.com/GoogleChrome/webrtc/pull/277 (first outdated comment).

**Pull request merged**